demo-ui/app/uk/gov/ons/addressIndex/demoui/views/singleMatch.scala.html

```diff
@@ -9,6 +9,27 @@
 
 @main(title = messages("single.title")) {
 <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.0/jquery.min.js"></script>
+
+    <script>
+        $(function () {
+           $(".formatted-nag-button").click(function (e) {
+               $(this).addClass("formatted-button-active");
```

Comment:
jquery is quite expensive
use:
```
var $this = $(this);
var $resultRow = $this.closest(".resultRow");
//etc
```

Also you can use `.toggle();` instead of `.show()` and `.hide()` on individual items